### ISSUE TITLE
testOffsetsForTimes fails

### ISSUE THREAD

**Issue by vietj** [vert-x3/vertx-kafka-client#22]
right now I'm getting:

```
testOffsetsForTimes(io.vertx.kafka.client.tests.ConsumerTest)  Time elapsed: 2.967 sec  <<< FAILURE!
java.lang.AssertionError: Expected true
	at io.vertx.ext.unit.impl.TestContextImpl.reportAssertionError(TestContextImpl.java:432)
	at io.vertx.ext.unit.impl.TestContextImpl.assertTrue(TestContextImpl.java:282)
	at io.vertx.ext.unit.impl.TestContextImpl.assertTrue(TestContextImpl.java:288)
	at io.vertx.kafka.client.tests.ConsumerTestBase.lambda$null$53(ConsumerTestBase.java:693)
	at io.vertx.core.impl.FutureImpl.checkCallHandler(FutureImpl.java:188)
	at io.vertx.core.impl.FutureImpl.tryComplete(FutureImpl.java:139)
	at io.vertx.core.impl.FutureImpl.complete(FutureImpl.java:106)
	at io.vertx.kafka.client.consumer.impl.KafkaReadStreamImpl.lambda$offsetsForTimes$28(KafkaReadStreamImpl.java:544)
	at io.vertx.kafka.client.consumer.impl.KafkaReadStreamImpl.lambda$submitTask$1(KafkaReadStreamImpl.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by doernemt** [comment]
@vietj I saw you checked in some changes Friday night. Is the problem solved or should I refactor the test further? 
Thx, Tim

**Comment by doernemt** [comment]
Fixed (indirectly) by #26
